src/include/enclave/enclave_util.h

```diff
@@ -138,10 +138,22 @@ extern int sgxlkl_trace_redirect_syscall;
         }                                                     \
     } while (0)
 
-#define SGXLKL_VERBOSE(x, ...)                                              \
-    if (sgxlkl_enclave_state.verbose)                                       \
-    {                                                                       \
-        oe_host_printf("[[  SGX-LKL ]] %s(): " x, __func__, ##__VA_ARGS__); \
+#define SGXLKL_VERBOSE(x, ...)                                               \
+    if (sgxlkl_enclave_state.verbose)                                        \
+    {                                                                        \
+        struct schedctx* _sgxlkl_verbose_self;                               \
+        __asm__ __volatile__("mov %%gs:48,%0" : "=r"(_sgxlkl_verbose_self)); \
```

Comment:
```suggestion
        struct schedctx* _sgxlkl_verbose_self = *ethread_schedctx;           \
```

All of the assembly for accessing this with hard-coded constants all over the place makes me nervous.  Let's add a global for this and clean up code to use that.  GCC and clang have good support for gs-relative addresses.

Reply:
Ok, I couldn't get this to work, as I presume that this needs gnu99 support? For this, we would also need to change musl to be compiled with gnu99...?